documentation/deliberately_vulnerable_flask_app/setup.sh

```diff
@@ -7,11 +7,10 @@
 # Source this file to run it:
 # . ./setup.sh
 
-# This file should ONLY include standard setup steps which would be used by a
-# real Flask project. Do not add anything Pysa/Pyre specific.
-
-python3 -m venv venv
-# shellcheck disable=SC1091
-. venv/bin/activate
-pip install Flask
-rm ../../.pyre_configuration
+#pip install -r requirements.txt
+#rm ../../.pyre_configuration
```

Comment:
Why are these commented? I would either uncomment them or remove them.

Reply:
I am a little surprised Pysa had as covered even though this was clearly a mistake.

Reply:
I am a little surprised Pysa had as covered even though this was clearly a mistake.